src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -55,7 +62,9 @@ struct GatherSlice
      */
     bool init(bool isActive)
     {
-        /*free old communicator of init is called again*/
+        static int masterRankOffset = 0;
```

Comment:
afaik it from normal static vars in functions: static variables in functions are only shared between following calls of the same function. if you really want to share it between all instances of `GatherSlice` is must be a static member type.